addons/mail/static/src/models/rtc.js

```diff
@@ -1190,12 +1184,19 @@ registerModel({
             if (!this.currentRtcSession.isMute) {
                 this.messaging.soundEffects.pushToTalkOff.play();
             }
-            this._pushToTalkTimeoutId = browser.setTimeout(
-                () => {
-                    this._setSoundBroadcast(false);
-                },
-                this.messaging.userSetting.voiceActiveDuration || 0,
-            );
+            this.update({
+                pushToTalkTimeout: this.messaging.browser.setTimeout(
+                    this._onPushToTalkTimeout,
+                    this.messaging.userSetting.voiceActiveDuration,
```

Comment:
Trailing comma in function parameters ?

Reply:
I works regardless. If we want to avoid accidental diffing from adding another positional parameter, then it's actually better to always add comma on last argument.